src/translation/translation_it_IT.ts

```diff
@@ -1194,7 +1194,7 @@ E&apos; stato disattivato l&apos;audio del tuo canale ed inserito il &quot;Disat
     <message>
         <location filename="../clientdlgbase.ui" line="343"/>
         <source>Delay</source>
-        <translation>Delay</translation>
+        <translation>Ritardo</translation>
```

Comment:
This should express something more like "Aspettare", I think - but I'm not sure of the contextual difference.
* "Se arrivi un po’ in ritardo non succede niente (if you arrive a little late, that’s OK)" from https://contentoitaliano.com/the-italian-verb-succedere-to-happen-to-occur/
* "Il gatto aspetta la cena – the cat is waiting for dinner" from https://contentoitaliano.com/what-does-aspettare-mean-in-italian/

So it's not that clear...  If it's definite that "Ritardo" is more appropriate for wall clock time delay, this is fine.

Reply:
"Ritardo" is fine. It's very similar in Spanish ("retardo"). In your first example, a literal translation would be "if you arrive with a little delay, that's OK".

Reply:
"quanto tempo aspetto" is "how long do I wait" - and the "delay", here, is how long you wait before the signal arrives, that's all.  "qual è il ritardo" - "what is the delay" - isn't quite the right meaning.  One is cause (ritardo), one is effect (aspetto).  But again, I'm not going to hold anyone to it as I don't know the language.  (It's just the English is vague.)

Reply:
You can see a few examples of how ritardare is used for delay [here](https://www.wordreference.com/enit/delay). If you say "qual è il ritardo"  I don't think it has the meaning of "what's causing the delay" but more literally "what is the delay" as in "what is the amount of time elapsing between x and y".